### DISCUSSION TITLE
News RSS feed

### DISCUSSION THREAD

**Discussion by avidseeker** [neovim/neovim#29059] [Q&A]
I'm following https://github.com/neovim/neovim/releases.atom, but the releases only include a link to https://neovim.io/doc/user/news-0.10.html

Is there a feed on neovim.io for the release notes?

**Comment by clason** [comment]
Those _are_ the release notes.

**Reply by avidseeker** [reply]
https://neovim.io/doc/user/news-0.10.html are the release notes for v0.10. I am looking for the release notes *feed*: RSS or Atom.

i.e: instead of subscribing to a feed that redirects me to other URL release notes, I want the feed for the release notes directly so that it can be viewed in my feed reader application.

**Reply by clason** [reply]
I see. Low priority, but PR to https://github.com/neovim/neovim.github.io welcome. (Note that this is not trivial since the documentation gets updated from `master` daily, so the "push" needs to be triggered some other way, e.g., via the release workflow here. Adding a small(!) step there would be acceptable, in principle.)

**Comment by justinmk** [comment]
You can follow any file in the repo by adding `.atom` to its `.../commits/master/...` URL:

https://github.com/neovim/neovim/commits/master/runtime/doc/news.txt.atom

**Reply by avidseeker** [reply]
Interesting. I didn't know about that!

**Comment by camoz** [comment]
There is also https://neovim.io/news.xml
